module/logic.js

```diff
@@ -18,11 +19,11 @@ class PartyOverviewApp extends Application {
 	}
 
 	update(ignoreNoActors = false) {
-		let actors = game.actors.contents
-			.filter((a) => a.hasPlayerOwner)
-			.map((playerActor) => playerActor.getActiveTokens())
-			.flat(1)
-			.map((token) => token.actor);
+		let showMoreTokens = false;
```

Comment:
This was a placeholder for a setting or something like that, right?

Reply:
Yes. If made some tests with that before introducing the SHOW_MORE mode.